cloudygames/tests.py

```diff
@@ -45,4 +57,44 @@ def test_get_owned_gamelist(self):
         client.force_authenticate(user=self.user2)
         response = client.get('/games/?owned=1')
         self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(len(response.data), 1)
+
+    def test_join_quit_game(self):
+        #user1 joins game1 & 2
```

Comment:
Some consideration to refactor the test code:
1. Separate each test into different methods
2. test one things and test it well.
3. use arrange-act-assert pattern.
4. use factory to construct the scenario instead of chain of actions.